corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -180,6 +188,11 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
         var mapping = {
             caption: {
                 update: function (options) {
+                    let parentForm = getParentForm(self);
+                    let oqps = parentForm.displayOptions.oneQuestionPerScreen != undefined && parentForm.displayOptions.oneQuestionPerScreen();
```

Comment:
Not your fault, but it's really confusing that the form's `displayOptions` properties are observables, while the user's `displayProperties` properties are plain attributes.

Reply:
yeah, that's definitely not ideal.